CLIENT_SPECIFICATION.md

```diff
@@ -0,0 +1,194 @@
+# tldr-pages client specification
+
+ - **Current Specification Version:** 2.0
+
+This document contains the official specification for tldr-pages clients. It is _not_ a specification of the format of the pages themselves - only a specification of how a user should be able to interface with an official client. 
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+
+## Changelog
+ - v1.0 (18th December 2018)
+	 - Initial draft
+ - v1.1 (18th December 2018)
+	 - Refined bullet points according to comments
+	 - Ensure the spec specifies the _interface_, not the workings
+ - v2.0 (12th January 2019)
+	 - Expanded notes into full sentences
+
+
+## Terminology
+In order to aid the understanding of this specification document, a number of terms will be defined in this section.
+
+### Page
+tldr-pages consists of multiple _pages_ - each of which describes a specific command.
+
+### Platform
+Pages are grouped by platform. A platform is an operating system. For example, `windows`, `linux`, `osx`. The special platform `common` contains pages for commands that work identically across all platforms.
+
+
+## Command-Line Interface
+This section describes the standardised command-line interface (CLI) for clients implementing one. Clients that do not provide a CLI can ignore this section.
+
+### Arguments
+A number of arguments MUST be supported (unless otherwise specified if a CLI is implemented:
+
+Argument		        | Meaning
+------------------------|--------------------
+`--update`				| Updates the offline cache of pages. MUST NOT be implemented if cache is not supported.
```

Comment:
how about
`Updates the offline cache of pages. MUST be implemented if cache is supported`